### ISSUE TITLE
Docstring of `mne.viz.plot_topomap()` incorrect

### ISSUE THREAD

**Issue by cbrnr** [mne-tools/mne-python#12698]
The docstring of `mne.viz.plot_topomap()` is incorrect. The descriptions of `contours` and `cmap` refer to an argument `colorbar` which does not exist:

https://mne.tools/stable/generated/mne.viz.plot_topomap.html

**Comment by deveshbervar** [comment]
Hello @cbrnr, if this issue is still open, I would like to work on it.

From my understanding, the docstring of `mne.viz.plot_topomap()` incorrectly refers to a non-existent `colorbar` argument, and fixing it would involve updating or removing those references in the descriptions of `cmap` and `contours`.

Could you please confirm if my understanding is correct and let me know if I can work on this issue? Thank you!

**Comment by cbrnr** [comment]
Exactly, although it only appears in the description of the `contours` argument. Thanks for working on it!
